package.json

```diff
@@ -43,14 +44,13 @@
     "@octokit/request-error": "^5.0.0",
     "@octokit/webhooks-methods": "^4.0.0",
     "@wolfy1339/openapi-webhooks-types": "5.1.3",
-    "aggregate-error": "^3.1.0"
+    "aggregate-error": "^5.0.0"
```

Comment:
While `AggregateError` was introduced natively in NodeJS v15, it isn't generic. So the types cannot be extended to use our custom error